packages/app/src/components/DimensionsPanel/styles/DndDimensionItem.module.css

```diff
@@ -2,7 +2,8 @@
 This keeps the dimension item clone from causing 
 a ripple effect in the Dimension list during drag
 */
-.dimensionItemClone ~ li {
+
+.notDragging {
     transform: none !important;
```

Comment:
Just a very small note, since this is a very specific use-case, but personally I'm not a huge fan of `!important` statements, is there possibly any way to solve this without it?

Reply:
Nobody is a fan of `!important` ;)

Unfortunately, this is one use case where it seems to be needed. This implementation is a "hack" into react-beautiful-dnd to get it to behave in a way that it doesn't do out of the box.